modules/oe_theme_helper/src/TwigExtension/TwigExtension.php

```diff
@@ -258,11 +258,13 @@ public function toEclAttributes($attributes): array {
    *   The twig context.
    * @param string|null $icon
    *   The icon to be converted.
+   * @param string $size
+   *   The icon size.
    *
    * @return array
    *   Icon array for ECL components containing icon name, path and rotation.
    */
-  public function toEclIcon(array $context, $icon): array {
+  public function toEclIcon(array $context, $icon, string $size = ''): array {
```

Comment:
https://ecl-twig-js.netlify.app/ec/?path=/story/components-icon--branded I see there are 3 optional elements but only size was added. Should we add also the others or size wouldn't be needed neither?